flat/ChildChainManager.sol

```diff
@@ -108,8 +108,8 @@ contract Initializable {
 
     modifier initializer() {
         require(!inited, "already inited");
-        inited = true;
         _;
+        inited = true;
```

Comment:
Why changed the order? We should set first just to avoid reentrancy.

Reply:
`RootChainManager` inherits `EIP712Base`, both are initializable.
RootChainManager initializer calls EIP712Base initializer.
Both these check the same `inited` variable. 
If `inited` is set before, then check fails in `EIP712Base`.

Reply:
These functions will be called only by us while deploying, can we assume it safe?